### PR TITLE
[#10] update google analytics

### PR DESCRIPTION
**Pull request by xhr15**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -47,7 +47,7 @@
         <p class="ug-info triangle-border left">
           ReactJS.berlin is a usergroup focused on ReactJS and related
      topics. We meet regularly on the 2nd Monday each month at 7p.m.
-          <a href="http://bitcrowd.net" title="bitcrowd.net Office">@bitcrowd Office</a>,
+          <a href="http://bitcrowd.net" title="bitcrowd.net office">@bitcrowd Office</a>,
```
Comment (planktonic) [inline index.html]: Should we make the capitalisation consistent? The visible text is still '**O**ffice'

File: index.html [hunk 2]
```diff
@@ -162,17 +162,26 @@
             For details on what kinds of behaviour are not tolerated and consequences for violating these rules, we refer to the <a href="http://rubyberlin.github.io/code-of-conduct">Berlin Code of Conduct</a>.
           </p>
         </div>
+        <div>
+          <h2>Privacy Policy and leagal</h2>
+          <p>
+            This site is run by
+            <a href="http://bitcrowd.net" title="bitcrowd.net software development for startups">@bitcrowd</a>.
+            You can find all leagal information <a href="http://bitcrowd.net" title="bitcrowd.net software development legal">here</a>.
```
Comment (planktonic) [inline index.html]: Perhaps make 'all legal information here' the link, instead of just 'here' — it’s more descriptive

Comment (planktonic) [reply index.html]: And spelling of 'legal' — only one 'a'

**Review by planktonic — COMMENTED** [review]
A couple of small questions & suggestions, otherwise all looks good

File: index.html (hunk 2 above)
Comment (xhr15) [reply index.html]: argh

File: index.html (hunk 1 above)
Comment (xhr15) [reply index.html]: Good point, I'will make it all lowercase

**Comment by xhr15** [comment]
@planktonic can haz look if good and merge if yes?

**Comment by xhr15** [comment]
I take care of the merge conflicts

**Comment by xhr15** [comment]
@planktonic: ready for final review

File: index.html [hunk 3]
```diff
@@ -150,32 +150,37 @@
...
+              hang out together, chat, listen to talks, exchange ideas and make new friends.
+              Any harmful or discriminating behaviour will not be tolerated and results
+              in the offending person being expelled from the meetup.
+            </p>
+            <p>
+              For details on what kinds of behaviour are not tolerated and consequences for violating these rules, we refer to the <a href="http://rubyberlin.github.io/code-of-conduct">Berlin Code of Conduct</a>.
+            </p>
+          </div>
+          <div>
+            <h2>Privacy Policy and leagal</h2>
```
Comment (planktonic) [inline index.html]: Ah, I missed this one — 'legal' again

**Review by planktonic — COMMENTED** [review]
Spotted one more spelling error, otherwise good to merge

File: index.html (hunk 3 above)
Comment (xhr15) [reply index.html]: noooo

**Pull request merged**